### PR TITLE
Handle countertops

### PR DESCRIPTION
**Pull request by influffity**
Add countertops and linters to machine management

### CODE REVIEW

File: levels/base_level/machine_manager.gd [hunk 1]
```diff
@@ -47,8 +45,7 @@ func _refresh_machines() -> void:
 
 # TODO: Properly connect to purchasing
 func _on_purchase(purchase_info: Dictionary) -> void:
```
Comment (metcalf-putnam) [inline levels/base_level/machine_manager.gd:47]: currently the purchase is done in "_on_item_purchased" and the item key is passed, not the dictionary for that item. Planning on having "_on_item_purchased" call "_on_purchase"? After first retrieving the dictionary for that item key? Seems like having two functions with very similar names would be confusing

**Review by metcalf-putnam — COMMENTED** [review]
Seems good, but a little confused about the _on_purchase structure and logic and how you plan on integrating that. Left specific comments in line

File: models/Ticket/sprite.png.import [hunk 2]
```diff
@@ -31,4 +31,4 @@ process/invert_color=false
 stream=false
 size_limit=0
 detect_3d=false
-svg/scale=1.0
+svg/scale=1.0
```
[the remark this answers is not in the document]
Comment (influffity) [reply models/Ticket/sprite.png.import:34]: It shows up as "Ticket" on github

File: game_utils/game_logic.gd [hunk 3]
```diff
@@ -28,6 +28,19 @@ const DRYERS = {
...
+const LINTERS = {
+  "basic_linter": {
+    "time": 5.0,
+    "level": 1
+  }
+}
+
+const COUNTERS = {
+  "basic_counter": {
+    "level": 1
```
[the remark this answers is not in the document]
Comment (influffity) [reply game_utils/game_logic.gd:40]: sounds good

**Pull request merged**